boofuzz/sessions.py

```diff
@@ -365,13 +370,20 @@ def __init__(self, session_filename=None, index_start=1, index_end=None, sleep_t
         self.restart_interval = restart_interval
         self.web_port = web_port
         self._keep_web_open = keep_web_open
+        self.console_gui = console_gui
         self._crash_threshold_node = crash_threshold_request
         self._crash_threshold_element = crash_threshold_element
         self.restart_sleep_time = restart_sleep_time
         if fuzz_data_logger is not None:
             raise exception.BoofuzzError('Session fuzz_data_logger is deprecated. Use fuzz_loggers instead!')
         if fuzz_loggers is None:
-            fuzz_loggers = [fuzz_logger_text.FuzzLoggerText()]
+            if self.console_gui and os.name != 'nt':
```

Comment:
Let's move the if statement up and out of `if fuzz_loggers is None` -- they seem like two separate possibilities to me. If we put the `console_gui` check first, then if it triggers, it will cause `if fuzz_loggers is None` to always be false.

I think the console gui should eventually be default.

Reply:
Good point, but we have to consider the case where a logger class is passed into `fuzz_loggers`.
This requires appending the FuzzLoggerCurses class to the passed loggers.
Should be doable, I'll test this.